SamanthaLHC/ft_transcendence
# Chat messages

This PR adds :
- the possibility to send a message to a channel
- the channel name is displayed when you are on the channel room
- if you have not joined the channel, you can't see the content or send messages

To test:
- Connect with your username and with test.
- With your user: Create a channel and go to it (click on its name). Write a message. It should appear.
- With test user: The channel shouldn't appear. Search it (search bar) ang go to it. You shouldn't see any message. Try to write a message.
- With your user: The message typed by test shouldn't appear.
- In prisma, UserChannelMap, add a record with test in your channel.
- Now test user should be able to see the channel and its content.
- Do the same with new channel.
- Messages from one channel shouldn't appear in the other channel.
- Type a lot of messages. The last message should appear just above the input field and the message field should not expand.
